assets/shaders/array_texture.wgsl

```diff
@@ -41,7 +41,7 @@ fn fragment(
     pbr_input.N = normalize(pbr_input.world_normal);
 
 #ifdef VERTEX_TANGENTS
-    let Nt = textureSampleBias(pbr_bindings::normal_map_texture, pbr_bindings::normal_map_sampler, mesh.uv, view.mip_bias).rgb;
+    let Nt = textureSampleBias(pbr_bindings,:: normal_map_texture, pbr_bindings,:: normal_map_sampler, mesh.uv, view.mip_bias).rgb;
```

Comment:
this doesn't look right

Reply:
Bug in wgslfmt. I guess this is a blocker?

Reply:
Yeah, this is not mergeable due to this bug.